crates/sui-data-ingestion-core/src/worker_pool.rs

```diff
@@ -160,7 +160,7 @@ impl<W: Worker + 'static> WorkerPool<W> {
 
         // Wait for all workers to finish
         for join_handle in join_handles {
-            join_handle.await.expect("worker thread panicked");
+            let _ = join_handle.await;
```

Comment:
why do we need this, presumably if workers exit gracefully due to `cancel`, the behavior before & after is the same?

Reply:
That wasn't needed. Forgot to remove. Thanks!